regolith/helpers/l_currentappointmentshelper.py

```diff
@@ -76,7 +76,7 @@ def sout(self):
         for person in people:
             p_appt = person.get("appointments", None)
             if p_appt:
-                for _id, appt in p_appt.items():
+                for k, (_id, appt) in enumerate(p_appt.items()):
```

Comment:
I don't think this is the right fix.  That k should be returning the key of the appointment, something like `appt.get("_id")` but it could be something else.  We don't want just an index there.

While we work to fix this, please can you also replace with f-strings.